crates/pglt_analyser/src/lint.rs

```diff
@@ -1,4 +1,4 @@
 //! Generated file, do not edit by hand, see `xtask/codegen`
 
 pub mod safety;
-::pglt_analyse::declare_category! { pub Lint { kind : Lint , groups : [self :: safety :: Safety ,] } }
+::pglt_analyse::declare_category! { pub Lint { kind : Lint , groups : [self :: safety :: Safety , ] } }
```

Comment:
maybe somebody accidentally typed `saftey` instead of `safety` and had to manually remove all the safteys from the code base and maybe that caused some white space diffs 🤓